app/assets/javascripts/lifeitup_layout.js

```diff
@@ -61,7 +61,10 @@ function layout_resizer () {
 // init popovers
 
 $(function () {
-  $('[data-toggle="popover"]').popover()
+  $('body').popover({
+    selector: '[data-toggle="popover"]',
+    trigger: 'focus'

```

Comment:
Yes. The lifeitup file should be moved to the vendor directory too (but that's most probably a matter of another PR). Since @cyntss is the one that knows lifeitup better, I believe that's better to just merge this and then she'll fix it later on (she has to fix some styling from this PR too anyways).

Hope that's ok for you @cyntss ?